modules/ossm-about-concepts-resources.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+// about/ossm-about-concepts-assembly.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-concepts-resources_{context}"]
+= {SMProductName} resources
+
+//Should each subject/topic be its own module in the about-concepts-assembly? Kiali module, traces modules, metrics module, Argo roll outs module, cert-manager module?
+//very rough draft, copy/pasted from upstream to create the PR to have a visual of what just upstream looks like in prod docs.
+//need Kiali, tracing, metrics, cert-manager, and Argo rollouts
+
+{SMProductName} Operator manages the lifecycle of your Istio control planes. Instead of creating a new configuration schema, {SMProduct} Operator APIs are built around Istio's Helm chart APIs.
+
+//All installation and configuration options that are exposed by Istio's Helm charts are available through the {SMProduct} Operator Custom Resource Definition (CRD) `values` fields.
+
+[NOTE]
+====
+* Though {SMProductName} APIs are built around Istio's Helm chart APIs, Helm charts are not supported.
+* All installation and configuration options that are exposed by Istio's Helm charts are available through the {SMProductName} Custom Resource Definition (CRD) `values` fields.
+* CRDs are supported.
+====
+
+== `Istio` resource
+
+The `Istio` resource is used to manage your Istio control planes. It is a cluster-wide resource, as the Istio control plane operates in and requires access to the entire cluster.
+
+To select a `namespace` to run the control plane pods in, you can use the `spec.namespace` field. `.
+
+[NOTE]
+====
+The `spec.namespace` field is immutable: in order to move a control plane to another namespace, you have to remove the `Istio` resource and recreate it with a different `spec.namespace`.
+====
+
+You can access all `Istio` custom resource defintion (CRD) options through `spec.values` fields:
```

Comment:
I think it'd be worth mentioning here that you can use `oc explain istios.spec.values` to see all the customization options.